pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/search/ElasticSearchParticipantDto.java

```diff
@@ -68,22 +68,23 @@ protected ElasticSearchParticipantDto() {  }
     /**
      * Changes the value for the given question's answer.
      * Does not make any modification to underlying elastic data.
+     * @return true if the value was changed, false if the activity
+     * and question do not exist.
      */
     @VisibleForTesting
-    public void changeQuestionAnswer(String activityCode, String questionStableId, String value) {
+    public boolean changeQuestionAnswer(String activityCode, String questionStableId, String value) {
```

Comment:
The way adult consent for tumor is determined requires for PE-CGS requires looking for either an OS-specific answer or an LMS-specific answer.  Rather than change the queries in the manifest test to be aware of LMS/OS study guids, I changed this method to return whether or not the field is found.  Existing clients are tests that will fail for various reasons if the expected fields are not set, so I've left those calls alone.  Effectively, nothing is checking this value as of now, and I think that's okay.